### DISCUSSION TITLE
Running a script in docker environment

### DISCUSSION THREAD

**Discussion by jayeshbadwaik** [community/community#27167] [Actions]
<p>I’ve searched quite widely, but I cannot seem to find an answer, hence I’m asking this here.</p>
<p>For a CI job, I want to run a sequence of scripts inside a custom docker container provided at <code>username/image:version. In a </code>gitlab` CI, this looks like follows:</p>
<pre><code class="lang-auto">format:
  image: user/image:version
  script:
    - ci/bin/format
    - ci/bin/post_results 
</code></pre>
<p>How can I achieve this in github actions?</p>
<p>Thank you</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Most likely by specifying a job container with <a href="https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainer"><code>jobs.&lt;job_id&gt;.container</code> </a>. Your <code>run</code> steps will then run in that container.</p>
